src/components/components/ebay-dialog-base/component.js

```diff
@@ -23,10 +23,14 @@ module.exports = {
         this.clickTarget = el;
     },
 
-    getActiveElement() {
+    getActiveElement(input) {
+        let closeFocusEl;
+        if (input && input.closeFocus) {
+            closeFocusEl = document.getElementById(input.closeFocus);
+        }
         const el =
             document.activeElement === document.body ? this.clickTarget : document.activeElement;
-        return el;
+        return closeFocusEl || el;
```

Comment:
If the `closeFocus` exists you do not have to get `el` so oyu can put that in an `else`

Reply:
No we want to default it to whatever was clicked.
It only defaults to whatever was clicked when closeFocus is not set or doesn't resolve to an element.

Reply:
I got it. I suggest:
```
if (closeFocusEl) return closeFocusEl;
return document.activeElement === document.body ? this.clickTarget : document.activeElement;
```
Instead of:
```
const el =
            document.activeElement === document.body ? this.clickTarget : document.activeElement;
        return closeFocusEl || el;
```

Reply:
By the way I am not sure what is the style for dev warn/error but it would be definitely useful if you have: 
```
        if (input && input.closeFocus) {
            closeFocusEl = document.getElementById(input.closeFocus);
        } else {
            console.warn('closeFocus id not found');
        }

Reply:
So the reason is this:
We want to make sure this goes to a happy path. If someone accidentally messes up, or an element isn't there, we want to default to the last click target. This is expected for dialog. We do not want this to lose focus. 
If the closeFocus is null, or if it cannot resolve to an element, it should default to the last click target. This is why the code is written like this because if the `document.getElementById` does not match, it returns null. So it will default to the click target.

I don't really want to warn, because users should be testing this before going out. I don't think this is necessary...